subsys/mgmt/ec_host_cmd/ec_host_cmd_handler.c

```diff
@@ -19,7 +19,7 @@
 #define TX_HEADER_SIZE (sizeof(struct ec_host_cmd_response_header))
 
 /** Used by host command handlers for their response before going over wire */
-uint8_t tx_buffer[CONFIG_EC_HOST_CMD_HANDLER_TX_BUFFER];
+static uint8_t tx_buffer[CONFIG_EC_HOST_CMD_HANDLER_TX_BUFFER] __aligned(8);
```

Comment:
Worth mentioning the alignment requirement in the comment above.

Reply:
Looks like it was an alignment from the previous version, it shouldn't be needed anymore

Reply:
Yea, you're right, I will...